packages/outputarea/test/widget.spec.ts

```diff
@@ -376,6 +376,14 @@ describe('outputarea/widget', () => {
           const factory = new OutputArea.ContentFactory();
           const future = kernel.requestExecute({ code: CODE });
           const options = {
+            parent_header: {
+              date: '',
+              msg_id: '',
+              msg_type: 'input_request',
```

Comment:
Not sure why `msg_type: 'input_request'` generates this error:
```
Error: test/widget.spec.ts(391,38): error TS2345: Argument of type '{ parent_header: { date: string; msg_id: string; msg_type: string; session: string; username: string; version: string; }; prompt: string; password: boolean; future: IShellFuture<IExecuteRequestMsg, IExecuteReplyMsg>; }' is not assignable to parameter of type 'IOptions'.
  The types of 'parent_header.msg_type' are incompatible between these types.
    Type 'string' is not assignable to type '"input_request"'.
```
I know it's related to [StdinMessageType](https://github.com/jupyterlab/jupyterlab/blob/ffbb016cd8d072654d00285bf0404cafa627a476/packages/services/src/kernel/messages.ts#L214), but I don't know why it is not accepted.